style.css

```diff
@@ -116,3 +116,348 @@
   gap: 19px;
   align-items: baseline;
 }
+
+#work-container {
+  display: grid;
+  flex-direction: column;
+  gap: 20px;
+  padding-top: 12px;
+  background-color: #fff;
+}
```

Comment:
- [ ] As the previous reviewer requested you need to change the background. It's good that you have changed the background :+1:. But It's not visible. :cry: Kindly change the background color according to the instructions

![background-color-fix](https://user-images.githubusercontent.com/93039359/176936384-7e37dbf0-a6e8-4a45-a3ea-4858537d72db.png)

Reply:
@HAMMAS-SALEEM Thank you for this thorough review. I will fix it now.